common/realsense-ui-advanced-mode.h

```diff
@@ -102,8 +102,17 @@ inline void slider_int(std::string& error_message, const char* id, T* val, S T::
             }
             else
             {
-                val->*field = static_cast<S>(new_value);
-                to_set = true;
+                if ((new_value > max) || (new_value < min))
+                {
+                    std::stringstream ss;
+                    ss << "New value " << new_value << " to be within [" << min << ", " << max << "] range";
+                    error_message = ss.str().c_str();
```

Comment:
`error_message` is a string -- why `.c_str()`?

Reply:
and we can drop the ` range` at the end

Reply:
Done